src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -2189,7 +2189,7 @@ namespace Microsoft.FSharp.Core
         /// <param name="x">The input value.</param>
         /// <returns>The same value.</returns>
         [<CompiledName("Identity")>]
-        val id : x:'T -> 'T 
+        val inline id : x:'T -> 'T 
```

Comment:
that one is important ;-)
But would it be automatically inlined because of beta reduction?